runtime/src/main/java/org/corfudb/runtime/view/LayoutBuilder.java

```diff
@@ -323,36 +335,52 @@ public LayoutBuilder removeSequencerServers(Set<String> endpoints)
         return this;
     }
 
+    /**
+     * Remove an endpoint from a segment.
+     *
+     * @param endpoint The endpoint to remove
+     * @param layoutStripe the stripe to remove the endpoint from
+     * @param mode mode of this segment
+     * @throws LayoutModificationException
+     */
+    public void removeFromStripe(String endpoint, LayoutStripe layoutStripe,
```

Comment:
Perhaps it should be a parameter here (minReplicationFactor)l

Reply:
Yes.